src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { getReviews } from "./utils/axios";
+import Header from "./components /Header/Header";
+import Review from "./components /Reviews/Review";
 
 function App() {
+  const [reviews, setReviews] = useState([]);
+  useEffect(() => {
+    getReviews().then((reviews) => {
+      setReviews(reviews);
+    });
+  }, []);
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      {reviews.map((review) => (
+        <Review review={review} />
+      ))}
```

Comment:
I'd recommend moving reviews into it's own component otherwise our App.js is going to get messy fast - remember with state we want to place it as low as possible, if we don't need reviews throughout our whole app it's overkill to put it at the top